modules/postfix/files/main.cf

```diff
@@ -46,3 +46,5 @@ notify_classes = delay, resource, software
 smtpd_sender_login_maps = ldap:/etc/postfix/ldap/smtpd_sender_login_maps
 smtpd_sender_restrictions = reject_sender_login_mismatch
 strict_mailbox_ownership = no
+
+header_checks = regexp:/^X-Spam-Level: [^*]*[*]{7,}$/ DISCARD
```

Comment:
For consistency, this should be done in a separate file similar to the LDAP protocol ones above rather than directly in the file.

Reply:
Sure, I'll make the change later. I thought about that also, but decided this way as if avoided a second file, but I understand the need for consistency here, so that is fine also. I'll make the change later. Thanks!

Reply:
Done.